From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

PRmerger commented:
@alexandair : Thanks for your contribution to the Azure documentation! The author, @rishirsinha, has been notified to review your proposed change.

rishirsinha commented:
Thanks for the contribution, @alexandair. @tysonn, please merge the PR.

alexandair commented:
@tysonn Could you merge this one, please?

alexandair commented:
@tysonn This one is approved for merging. :) I also have a few PRs opened for more than 2 weeks. Could someone ping the authors, please?

tysonn commented:
Hi @alexandair - Thanks for the prompt. I couldn't find a notification for this one in my mail folders, but they seem to be working today. We're working on automation and reporting to help improve people's response to contributions. In the meantime I periodically send manual reports to try to drive responses.

cephalin committed:
add auth tutorial

cephalin committed:
acrolinx

cephalin committed:
acrolinx

cephalin committed:
nits

cephalin committed:
shorten headings

cephalin committed:
nits

TomArcherMsft committed:
Initial draft

TomArcherMsft committed:
Acrolynx pass and image renaming

TomArcherMsft committed:
Acrolynx pass 2

TomArcherMsft committed:
Acrolynx pass 3

TomArcherMsft committed:
Acrolynx pass 4 and image name correction

TomArcherMsft committed:
Added 'Next steps' section

TomArcherMsft committed:
Reworking intro and first steps

TomArcherMsft committed:
Adding steps

TomArcherMsft committed:
Adding steps 2

TomArcherMsft committed:
Fixed formatting

Commit:
first cut

Commit:
fixed Cloud Shell include link

Commit:
fixes

Commit:
fixes/acrolinks

TomArcherMsft committed:
Working on steps to provision infraqstructure

Commit:
more edits

Commit:
acrolynx checks

Commit:
automation : correcting service limits

Commit:
ticks around clouddrive and fixed up back-end/front-end

rwike77 committed:
added console redirection

TomArcherMsft committed:
Finished first section

TomArcherMsft committed:
Added Prerequisites section

TomArcherMsft committed:
GitHub section

TomArcherMsft committed:
Deployment slot section

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sf-linux-container-qs-fix
sync with upstream

rwike77 committed:
update

rwike77 committed:
updated default policies

TomArcherMsft committed:
Testing

Commit:
fixed images and fixed instructions to be performed from Voting directory

Commit:
re-added initial image

rwike77 committed:
updated local user and user group sections, added examples

rwike77 committed:
typo

Commit:
removed green checks and added Next link

Commit:
Update instructions for party cluster certs

Commit:
get rid of checkmarks and add single step for Next steps per MVC

Commit:
drag n drop instead of clouddrive

Commit:
small fix

Commit:
specify PEM file in openssl fingerprint command

cephalin committed:
matt feedback

cephalin committed:
acrolinx

Commit:
Fixes for versions of Eclipse later than Neon and added installation instruction for Linux dev environment

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into hrwupdates

billmath committed:
staging marks updates

Commit:
hrw updates

TomArcherMsft committed:
Last sections

TomArcherMsft committed:
Acrolynx pass, image deletion

TomArcherMsft committed:
Test pass - cleaning up

Commit:
updates

TomArcherMsft committed:
Test pass - cleaning up 2

TomArcherMsft committed:
Test pass - cleaning up 3

TomArcherMsft committed:
Modified images

TomArcherMsft committed:
Word smithing

TomArcherMsft committed:
Reshot image

TomArcherMsft committed:
More cleanup during run-through

TomArcherMsft committed:
Image work

TomArcherMsft committed:
Wording smithing to make steps easier to follow

TomArcherMsft committed:
More word tweaking

TomArcherMsft committed:
More image tweaking

TomArcherMsft committed:
Renumbered steps

TomArcherMsft committed:
Trying to fix image

TomArcherMsft committed:
More steps tweaking

TomArcherMsft committed:
Working on last section

TomArcherMsft committed:
Corrected styling

TomArcherMsft committed:
Corrected styling 2

rwike77 committed:
called out upgrade progress in screenshot

rwike77 committed:
called out upgrade progress in screenshot

rwike77 committed:
made clarifications

cherylmc committed:
update

cherylmc committed:
update

cherylmc committed:
update

rwike77 committed:
moved note

cherylmc committed:
update

cherylmc committed:
includes

TomArcherMsft committed:
nightly commit

MohitGargMSFT committed:
(ForumsTeam) Fixes MicrosoftDocs/azure-docs#6709

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#6709

Commit:
6.2 ai update

TomArcherMsft committed:
Working on last section

TomArcherMsft committed:
First draft?

TomArcherMsft committed:
styling issue

Commit:
updates

TomArcherMsft committed:
remove summary section

cephalin committed:
toc change

cephalin committed:
Merge branch 'master' into auth

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into hrwupdates

Commit:
automation: corrections for issue 6766

TomArcherMsft committed:
Changes per tech review

jpconnock committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
updates to text

Commit:
added hashes

cherylmc committed:
fix

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into includefix

cherylmc committed:
update

CarlRabeler committed:
Update sql-database-long-term-backup-retention-configure.md

TomArcherMsft committed:
Fixes

Commit:
updates to verbiage

Commit:
added collector hashes

Commit:
Merge pull request #37136 from CarlRabeler/patch-50

Update sql-database-long-term-backup-retention-configure.md

Commit:
changed overview

Commit:
changed 1000

TomArcherMsft committed:
Image work

Commit:
addressed Deep's comments

rwike77 committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into securityfixes

rwike77 committed:
removed NTLM security info

jpconnock committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

GitHubber17 committed:
Merge pull request #37135 from ruturaj/master

added hashes

rwike77 committed:
removed username from image

GitHubber17 committed:
Merge pull request #36367 from TomArcher/terraform-slot-walkthru

Terraform Azure deployment slot walkthru

Ja-Dunn committed:
Merge pull request #37095 from srrengar/6.2docupdateAI

6.2 ai update

mikeurnun committed:
fixes #6779

fixes MicrosoftDocs/azure-docs#6779

mimig1 committed:
Cosmos DB: Moving Mimi's topics to Sneha&Kelly

GitHubber17 committed:
Merge pull request #37074 from rwike77/tutorialfixes

.net tutorial usability fixes

GitHubber17 committed:
Merge pull request #36669 from rwike77/securityfixes

added console redirection

mimig1 committed:
Cosmos DB: Moving Mimi's files to Sneha & Kelly (2)

PRMerger12 committed:
Merge pull request #37094 from MohitGargMSFT/patch-11

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#6709

cephalin committed:
prmerger feedback

MarkMcGeeAtAquent committed:
fix

mimig1 committed:
Cosmos DB: Moving Mimi's files to Sneha & Kelly (3)

carolinacmoravia committed:
L15618: Extra space between square bracket and parenthesis

Hello, @jimdial, 

Localization team has reported source content issue that causes localized version to have broken/different format compared to en-us version.  

<Provide further explanation here as needed>  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

GitHubber17 committed:
Merge pull request #36087 from cephalin/auth

add auth tutorial

PRMerger18 committed:
Merge pull request #37147 from MarkMcGeeAtAquent/php-fix

Cosmos DB: removed alert re: batch operations (which are now fixed)

Commit:
Merge pull request #37144 from mimig1/04052018meta

Cosmos DB: Moving Mimi's topics to Sneha&Kelly

Commit:
Merge pull request #37143 from mike-urnun-msft/patch-4

fixes #6779

TomArcherMsft committed:
corrected images

anirudhcavale committed:
Multi-Dim metric warning

anirudhcavale committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into DiagSettingMultiDim

GitHubber17 committed:
Merge pull request #37153 from TomArcher/terraform-slots-fix-image

Correcting image in Deployment Slots article

tfitzmac committed:
added event grid sub queries

mimckitt committed:
Fixes MicrosoftDocs/azure-docs#6777

Missing -MiniProtocolVersion XXXX

Set-AzureRmApplicationGatewaySSLPolicy -MinProtocolVersion -PolicyType Custom -CipherSuite "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256", "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384", "TLS_RSA_WITH_AES_128_GCM_SHA256" -ApplicationGateway $gw

Corrected to Set-AzureRmApplicationGatewaySSLPolicy -MinProtocolVersion TLSv1_2 -PolicyType Custom -CipherSuite "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256", "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384", "TLS_RSA_WITH_AES_128_GCM_SHA256" -ApplicationGateway $gw

GitHubber17 committed:
Merge pull request #37154 from anirudhcavale/DiagSettingMultiDim

Diag setting multi dim

jpconnock committed:
Updating manager metadata

jpconnock committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into JPmetadata1

GitHubber17 committed:
Merge pull request #37157 from tfitzmac/0404queryeventgrid

added event grid sub queries

mimckitt committed:
Fixes MicrosoftDocs/azure-docs#6749

The size tables have changed and the links for DS2 and DS3 were incorrect.

PRMerger20 committed:
Merge pull request #37128 from georgewallace/issue6766

automation: corrections for issue 6766

codemillmatt committed:
Merge remote-tracking branch 'MicrosoftDocs/master' into xam-cosmos-sql-qs-date-change

codemillmatt committed:
Updating the year of the article update

Commit:
Merge pull request #37149 from carolinacmoravia/patch-664

L15618: Extra space between square bracket and parenthesis

Commit:
Merge pull request #37145 from mimig1/04052018meta2

Cosmos DB: Moving Mimi's files to Sneha & Kelly (2)

Commit:
metadata for include

Commit:
Merge pull request #37160 from MicahMcKittrick-MSFT/patch-17

Fixes MicrosoftDocs/azure-docs#6749

JasonWHowell committed:
Update stream-analytics-build-an-iot-solution-using-stream-analytics.md

cherylmc committed:
update

mimig1 committed:
Cosmos DB: Moving Mimi's files to Sneha & Kelly (4)

rmca14 committed:
Merge pull request #37148 from mimig1/04052018meta3

Cosmos DB: Moving Mimi's files to Sneha & Kelly (3)

Commit:
Merge pull request #37162 from codemillmatt/xam-cosmos-sql-qs-date-change

Xam cosmos sql qs date change

mimig1 committed:
Cosmos DB: Moving Mimi's files to Sneha & Kelly (5)

rmca14 committed:
Merge pull request #37158 from MicahMcKittrick-MSFT/patch-16

Fixes MicrosoftDocs/azure-docs#6777

GitHubber17 committed:
Merge pull request #36627 from georgewallace/hrwupdates

automation : correcting service limits

tfitzmac committed:
fixes MicrosoftDocs/azure-docs#6561

mimig1 committed:
Cosmos DB: Moving Mimi's files to Sneha & Kelly (6)

cherylmc committed:
format

PRMerger13 committed:
Merge pull request #37163 from JasonWHowell/patch-2

Update stream-analytics-build-an-iot-solution-using-stream-analytics.md

mimig1 committed:
Cosmos DB: Moving Mimi's files to Sneha & Kelly (7).

PRMerger13 committed:
Merge pull request #37164 from mimig1/04052018meta4

Cosmos DB: Moving Mimi's files to Sneha & Kelly (4)

PRMerger14 committed:
Merge pull request #37003 from billmath/ac1

staging marks updates

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sf-linux-container-qs-fix
sync from upstream/master

PRMerger15 committed:
Merge pull request #36969 from JimacoMS2/sf-eclipse-fix

Fixes for versions of Eclipse later than Neon and added installation …

mimig1 committed:
Cosmos DB: Moving Mimi's files to Sneha & Kelly (8)

PRMerger15 committed:
Merge pull request #37161 from jpconnock/JPmetadata1

Updating metadata to identify correct manager

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sf-springboot-qs-fix
sync with upstream/master

PRMerger16 committed:
Merge pull request #36870 from JimacoMS2/win-qs-mvc-fixes

get rid of checkmarks and add single step for Next steps per MVC

PRMerger16 committed:
Merge pull request #37167 from mimig1/04052018meta6

Cosmos DB: Moving Mimi's files to Sneha & Kelly (6)

mimig1 committed:
Cosmos DB: Moving Mimi's files to Sneha & Kelly (9)

Commit:
Merge pull request #37168 from tfitzmac/0405egconcepts

added recommendation about topics

ktoliver committed:
Merge pull request #36561 from JimacoMS2/sf-linux-container-qs-fix

Linux container quickstart

tamram committed:
fixing container name - issue 6719

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

tfitzmac committed:
added PowerShell for sending event

ktoliver committed:
Merge pull request #36868 from JimacoMS2/sf-springboot-qs-fix

Java quickstart fixes. Add party cluster cert support and MVC compliance

rmca14 committed:
Merge pull request #37089 from cherylmc/includefix

Includefix

Brenduns committed:
fixing a ui reference that changed

gsacavdm committed:
Add Azure Advisor to list of services in Azure Gov

ajlam committed:
Merge pull request #6 from MicrosoftDocs/master

Update fork

ajlam committed:
Update MySQL and PG scale sample

gsacavdm committed:
Add Azure Advisor variations for gov

PRMerger15 committed:
Merge pull request #37169 from mimig1/04052018meta7

Cosmos DB: Moving Mimi's files to Sneha & Kelly (7).

PRMerger15 committed:
Merge pull request #37177 from Brenduns/april-02

fixing a ui reference that changed

PRMerger16 committed:
Merge pull request #37175 from tamram/tamram-6719

fixing container name - issue 6719

v-alje committed:
Merge pull request #37181 from MicrosoftDocs/FromPublicRepo

From public repo

cherylmc committed:
Update vpn-gateway-vpn-faq.md

PRMerger17 committed:
Merge pull request #37171 from mimig1/04052018meta9

Cosmos DB: Moving Mimi's files to Sneha & Kelly (9)

PRMerger17 committed:
Merge pull request #37180 from ajlam/master

Update MySQL and PG scale sample

ktoliver committed:
Merge pull request #37176 from tfitzmac/0405portal

added PowerShell for sending event

Ja-Dunn committed:
Merge pull request #37165 from mimig1/04052018meta5

Cosmos DB: Moving Mimi's files to Sneha & Kelly (5)

Ja-Dunn committed:
Merge pull request #37170 from mimig1/04052018meta8

Cosmos DB: Moving Mimi's files to Sneha & Kelly (8)

PRMerger20 committed:
Merge pull request #37184 from cherylmc/patch-2

Update vpn-gateway-vpn-faq.md

PRMerger20 committed:
Merge pull request #37178 from gsacavdm/patch-1

Add Azure Advisor to list of services in Azure Gov
